sszgen/tagparse_test.go

```diff
@@ -0,0 +1,141 @@
+package main
```

Comment:
Do bitlists need to be tested in here? or is everything you need to know contained in their name, e.g. bitvector4

Reply:
bitlists are picked up by the type name (this was also the case in the fastssz code before) but i did just also add support for the `ssz:"bitlist"` tag.